src/js/themes/hpe.js

```diff
@@ -338,50 +378,18 @@ export const hpe = deepFreeze({
       },
     },
     disabled: {
```

Comment:
I actually think we can remove this entire block of "disabled". `opacity: 0.3` comes from grommet theme (we can leave here to be explicit if we want) and the `undefined` overrides for button kinds/background/etc. were only there to override what we had in grommet-theme-hpe (which will now be removed).

Reply:
done.